src/renderer/services/ethereum/fees.ts

```diff
@@ -47,3 +49,32 @@ export const callFees$ = (
     RxOp.catchError((_) => Rx.of(RD.success(getDefaultFees()))),
     RxOp.startWith(RD.pending)
   )
+
+export const outTxFee$ = (asset: Asset): C.FeesLD =>
```

Comment:
Pls extend `createFeesService` of ETH to include `outTxFee$` + `callFee$`. Take `createTransactionService` of ETH as an example.